_episodes/XX-access-data.md

```diff
@@ -0,0 +1,329 @@
+associated to accurate geographic coordinate information.
+
+In this episode we will explore how to access open satellite data using Python. In particular,  we will
+consider [the Sentinel-2 data collection that is hosted on AWS](https://registry.opendata.aws/sentinel-2-l2a-cogs).
+This dataset consists of multi-band optical images acquired by the two satellites of
+[the Sentinel-2 mission](https://sentinel.esa.int/web/sentinel/missions/sentinel-2) and it is continuously updated with
+new images.
+
+# Search for satellite imagery
+
+## The SpatioTemporal Asset Catalog (STAC) specification
+
+Current sensor resolutions and satellite revisit periods are such that terabytes of data products are added daily to the
+corresponding collections. Such datasets cannot be made accessible to users via full-catalog download. Space agencies
+and other data providers often offer access to their data catalogs through interactive graphical user interfaces (GUIs,
+see for instance the [Copernicus Open Access Hub portal](https://scihub.copernicus.eu/dhus/#/home) for the Sentinel
+missions). Accessing data via a GUI is a nice way to explore a catalog and get familiar with its content, but it
+represents a heavy and error-prone task that should be avoided if carried out systematically to retrieve data.
+
+A service that offers programmatic access to the data enables users to reach the desired data in a more reliable,
+scalable and reproducible manner. An important element in the software interface exposed to the users (generally called
+the application programming interface, API) is the use of standards. Standards, in fact, can significantly facilitate
+the reusability of tools and scripts across datasets and applications.
+
+The SpatioTemporal Asset Catalog (STAC) specification is an emerging standard for describing geospatial data. By
+organizing metadata in a form that adheres to the STAC specifications, data providers make it possible for users to
+access data from different missions, instruments and collections using the same set of tools.
+
+> ## More Resources on STAC
+>
+> * [The STAC specification](https://github.com/radiantearth/stac-spec)
+> * [Tools based on STAC](https://stacindex.org/ecosystem)
+> * [STAC catalogs](https://stacindex.org/catalogs)
+{: .callout}
+
+## Search a STAC catalog
+
+The Sentinel-2 collection hosted on AWS is indexed in a STAC catalog that is accessible at the following link:
+
+~~~
+api_url = "https://earth-search.aws.element84.com/v0"
+~~~
+{: .language-python}
+
+> ## Exercise: Discover a STAC catalog
+> Open the following STAC API link using your web browser: https://earth-search.aws.element84.com/v0.
+> Navigate through the links to find out which collections are available and how many scenes are indexed. Where may one
+> find information on how to query the API for the desired scenes? Can you find out which parameters can be provided
+> in the queries?
+{: .challenge}
+
```

Comment:
I think we might want to show learners a graphical tool to browse STAC Catalogs. This one shows the spatial extent and summarizes the information about any STAC catalog url you paste into it.

https://radiantearth.github.io/stac-browser

Reply:
for example: https://radiantearth.github.io/stac-browser/#/external/earth-search.aws.element84.com/v0/collections/sentinel-s2-l2a

Reply:
Sounds great, and thanks for the tips - I hadn't seen yet the new STAC browser! Unfortunately the filtering tools do not seem to work with the Earth Search STAC API (maybe because this is an older STAC API version, 0.9), but [I have read ](https://medium.com/radiant-earth-insights/the-exciting-future-of-the-stac-browser-2351143aa24b) this is still a "demo" version, so things might be fixed soon. Anyway, for the purpose of the exercise, i.e. browsing through the items, works very well!